spec/forms/otp_delivery_selection_form_spec.rb

```diff
@@ -42,5 +42,32 @@
         expect(result).to eq result_hash
       end
     end
+
+    context 'when otp_method is the same as the user otp_delivery_preference' do
+      it 'does not update the user' do
+        user = build_stubbed(:user, otp_delivery_preference: 'sms')
+        form = OtpDeliverySelectionForm.new(user)
+
+        expect(UpdateUser).to_not receive(:new)
```

Comment:
I think in other parts of the test suite we've been moving to including `allow(thing).to receive(:method)` in the setup and than `expect(thing).to have_received(:method)` so that the expectations are the last part of the spec